xla/pjrt/pjrt_executable.h

```diff
@@ -160,7 +160,7 @@ struct PjRtTransferMetadata {
 };
 
 class PjRtChunk;
-class PjRtTransferMetadata;
+struct PjRtTransferMetadata;
```

Comment:
Thanks for fixing this, but I believe it should be possible to completely remove this line. The struct is defined just above.